### ISSUE TITLE
How to implement MASK RCNN on Android？

### ISSUE THREAD

**Issue by mengxiabing** [tensorflow/models#3632]
How to implement MASK RCNN on Android？

**Comment by mengxiabing** [comment]
或者用MobileNet实现

**Comment by mengxiabing** [comment]
Or implement it with MobileNet

**Comment by AliceDinh** [comment]
@mengxiabing : convert .h5 model to .pb model by this: https://github.com/amir-abdi/keras_to_tensorflow/blob/master/keras_to_tensorflow.py
Then use this: https://www.tensorflow.org/mobile/tflite/demo_android
to create your mobile app with MaskRCNN

**Comment by AliceDinh** [comment]
If you have any difficulties, refer to #218

**Comment by ColinWang1203** [comment]
> @mengxiabing : convert .h5 model to .pb model by this: https://github.com/amir-abdi/keras_to_tensorflow/blob/master/keras_to_tensorflow.py
> Then use this: https://www.tensorflow.org/mobile/tflite/demo_android
> to create your mobile app with MaskRCNN

We can convert the .h5 to .pb file, but putting it in tensorflow android app will not work right?
I suppose there is no mask drawing in it.
Is there any code on github managed to output the maskrcnn.pb result?

**Comment by kittusster** [comment]
any solution for this

**Comment by Aisharjaa** [comment]
We got a .pth file. How can we connect this to an app?or we should apply another process?

**Comment by Shayendra** [comment]
how to convert mask rcnn h5 custom data model to tflite and how to deploy it on android
